neurobooth_os/config.py

```diff
@@ -2,24 +2,31 @@
 """
     Ensures that the base neurobooth-os config file exists and makes config file available as neurobooth_config
 """
+import os.path
```

Comment:
Below line imports same library with alias of `op`

Reply:
fixed